indico/modules/events/contributions/views.py

```diff
@@ -25,7 +25,8 @@ class WPManageContributions(WPJinjaMixin, WPConferenceModifBase):
     sidemenu_option = 'contributions'
 
     def getJSFiles(self):
-        return (WPConferenceModifBase.getJSFiles(self) + self._asset_env['modules_contributions_js'].urls())
+        return (WPConferenceModifBase.getJSFiles(self) + self._asset_env['modules_contributions_js'].urls() +
+                self._asset_env['taggle_js'].urls())
```

Comment:
I think you need to include this on the WP used for meeting display too (or just include it in the global bundle) - you have an "edit contribution" dialog there after all